src/AtulS/CharacterCount.java

```diff
@@ -0,0 +1,31 @@
+//WAP to character occurring in user defined String and Character
+package AtulS;
+
+import java.util.Scanner;
+
+public class CharacterCount {
+
+	int findcharnew(String name, char ch, int index) {
```

Comment:
why we are passing 3rd param as index.. what is the use of it..

Reply:
To achieve recursion in this program, we require one variable which will be compared with string length. Hence we can define index as instance variable or function parameter.
Please correct me if i am wrong.

Reply:
Ok @AtulSAug19